src/shared/utils.js

```diff
@@ -55,7 +55,9 @@ const loadConfig = function (configFile) {
 
     return {
         server: {
-            port: process.env.SERVERPORT || config.server.port
+            port: process.env.SERVERPORT || config.server.port,
+            loginTokenExpiresIn: process.env.LOGINTOKENEXPIRESIN || config.server.loginTokenExpiresIn,
```

Comment:
It might be best at some point to namespace all of these env variables with a prefix since it can be helpful to be able to guess which application an env variable is targeted at as well as guarding against collisions by variable required in other environments, e.g. netlify - might it make sense to name them PDS_SERVERPORT, etc.?